lasso/emme.py

```diff
@@ -793,14 +797,266 @@ def prepare_table_for_taz_transit_network(
         links_df = pd.concat([links_df, new_link_gdf], sort = False, ignore_index = True)
         
         links_df.drop_duplicates(subset = ['A', 'B'], inplace = True)
+
+    ### PNR edits -- create PNR dummy links, pnr/knr/walk connectors, transfer links
```

Comment:
This is not generically emme, we should eventually move it to roadway.py or bart.py. Also, wold be good to split into separate methods instead of one big chunk.